.travis/opensuse/travis-tasks.sh

```diff
@@ -0,0 +1,115 @@
+#!/bin/bash
+
+#Exit on failures
+set -e
+
+set -x
+
+JOB_NAME=${TRAVIS_JOB_NAME:Opensuse tumbleweed}
+
+arr=($JOB_NAME)
+os_name=${arr[0]:-Opensuse}
+release=${arr[1]:-tumbleweed}
+
+COMMON_MESON_ARGS="-Dtest_dirty_git=${DIRTY_REPO_CHECK:-true}"
+
+
+pushd /builddir/
+
+# Build the v1 and v2 code under GCC and run standard tests
+meson --buildtype=debug \
+      -Dbuild_api_v1=true \
+      -Dbuild_api_v2=true \
+      $COMMON_MESON_ARGS \
+      travis
+
+set +e
+ninja -C travis test
+if [ $? != 0 ]; then
+    cat travis_v1/meson-logs/testlog.txt
```

Comment:
There was a bug in the centos test here that I fixed recently. Make the same fix here:
```suggestion
    cat travis/meson-logs/testlog.txt
```